.github/workflows/evalml_main_performance_tests.yaml

```diff
@@ -0,0 +1,139 @@
+      AWS_DEFAULT_REGION: 'us-east-1'
+      AWS_SESSION_TOKEN: ""
+    steps:
+      - name: Checkout evalml
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get commit hashes
+        id: get_hashes
+        run: |
+          current_hash=$(git rev-parse --short HEAD)
+          echo "Latest commit hash: $current_hash"
+          echo "::set-output name=current_hash::$current_hash"
+          previous_hash=$(git rev-parse --short HEAD~1)
+          echo "Previous commit hash: $previous_hash"
+          echo "::set-output name=previous_hash::$previous_hash"
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Checkout looking glass
+        uses: actions/checkout@v2
+        with:
+          repository: alteryx/looking-glass
+          ref: master
+          token: ${{ secrets.LOOKING_GLASS_TOKEN }}
+      - name: Installing Dependencies
+        run: |
+          make installdeps
+          make installdeps-test
+      - name: Run default algorithm performance test (current)
+        id: current_default
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo default --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.current_hash }} --job-desc ${{ steps.get_hashes.outputs.current_hash }}_default )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run iterative algorithm performance test (current)
+        id: current_iterative
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo iterative --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.current_hash }} --job-desc ${{ steps.get_hashes.outputs.current_hash }}_iterative )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run default algorithm performance test (previous)
+        id: previous_default
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo default --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.previous_hash }} --job-desc ${{ steps.get_hashes.outputs.previous_hash }}_default )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run iterative algorithm performance test (previous)
+        id: previous_iterative
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo iterative --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.previous_hash }} --job-desc ${{ steps.get_hashes.outputs.previous_hash }}_iterative )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Await results
```

Comment:
should we add a timeout here or is there a default timeout for github actions? If the ECS machines fall over, the status will never hit 'COMPLETED'.

Reply:
i'm assuming the jobs don't get marked as 'FAILED', so there's no way to know they died.

Reply:
I think six works for what we have (we could even lower it to 3 hours) but it could be insufficient if we ever need to run really large datasets.

Reply:
yup - thats true currently.